iconservice/icon_service_engine.py

```diff
@@ -662,10 +653,12 @@ def validate_transaction(self, request: dict) -> None:
 
         self._icon_pre_validator.execute(context, params, step_price, minimum_step)
 
-        self._validate_score_blacklist(context, params)
+        IconScoreContextUtil.validate_score_blacklist(context, to)
         if IconScoreContextUtil.is_service_flag_on(context, IconServiceFlag.DEPLOYER_WHITE_LIST):
             self._validate_deployer_whitelist(context, params)
```

Comment:
move function to IconScoreContextUtil
"_validate_deployer_whitelist"

remove function _validate_score_blacklist in iconserviceengine